app/mqtt.mjs

```diff
@@ -56,6 +59,16 @@ export const publishValues = async (modbusClient, mqttClient) => {
         topicMap[topicName] = JSON.stringify(value)
     }
 
+    const alarmStatuses = await getAlarmStatuses(modbusClient, false, true)
+
+    for (const [index, alarm] of Object.entries(alarmStatuses)) {
+        const topicName = `${TOPIC_PREFIX_ALARM}/${alarm.name}`
+
+        // Boolean values are changed to "ON" and "OFF" respectively since those are the
+        // defaults for MQTT binary sensors in Home Assistant
+        topicMap[topicName] = alarm.state > 0 ? 'ON' : 'OFF'
```

Comment:
If `0=OFF, 1=KUITATTU, 2=ON` from the documentation is true, then this should probably be `alarm.state == 2 ? 'ON' : 'OFF'`?

Reply:
1 to me is still an active, pending for off state, but if you want to have kuitattu as off, it is ok with me. Should I change?

Reply:
I'm not really sure what's right here to be honest. I would think that one acknowledges alarms only when they've been fixed.

Reply:
Looking in the control panels history, all my alarms there are in OFF state. However service reminder for example goes for a short while to dismissed after clearing the alarms that is 2 => 1 => 0, so I would assume that 1 means that we have dismissed it but it might go back to a 2 if the problem persists, or isn´t that the case? 

alarm.state == 2 ? 'ON' : 'OFF' is fine with me if you want it like that, most important is that 2 is ON and 0 is OFF :D